generic_permissions/visibilities.py

```diff
@@ -60,6 +60,30 @@ def get_attribute(self, instance):
         return queryset
 
 
+class VisibilitySerializerMixin:
```

Comment:
This would mean that all serializers using `VisibilityManyRelatedField` need to inherit from this mixin right? If so, we should add documentation for this.

Reply:
Added a line in the README.

> To prevent that include the VisibilitySerializerMixin in your serializer where you use ManyToMany relations.

Reply:
:+1: